res/skins/Tango/skin_settings.xml

```diff
@@ -246,17 +371,64 @@ Description:
                     </Connection>
                   </WidgetGroup>
 
-                  <WidgetGroup><!-- SpinnyCover submenu buttons -->
+                  <WidgetGroup><!-- small/BIG button without Channel mixer -->
                     <Layout>vertical</Layout>
-                    <Size>172f,30f</Size>
+                    <Size>182f,15f</Size>
                     <Children>
+                      <Template src="skin:skin_settings_button_2state.xml">
+                        <SetVariable name="state_0_text">      ° &#9664;  O</SetVariable>
+                        <SetVariable name="state_1_text">      °  &#9654; O</SetVariable>
+                        <SetVariable name="Setting">[Tango],spinnyCoverBIG</SetVariable>
+                      </Template>
+                    </Children>
+                    <Connection>
+                      <ConfigKey persist="true">[Tango],channel_mixer</ConfigKey>
+                      <BindProperty>visible</BindProperty>
+                      <Transform><Not/></Transform>
+                    </Connection>
+                  </WidgetGroup><!-- /small/BIG button without Channel mixer -->
 
-                    <Template src="skin:skin_settings_button_2state.xml">
-                        <SetVariable name="state_0_text">      ° &#9668; O</SetVariable>
-                        <SetVariable name="state_1_text">      ° &#9658; O</SetVariable>
+                  <WidgetGroup><!-- small/BIG button with Channel mixer -->
+                    <Layout>vertical</Layout>
+                    <Size>182f,15f</Size>
+                    <Children>
+                      <Template src="skin:skin_settings_button_2state.xml">
+                        <SetVariable name="state_0_text">      ° &#9664;  O</SetVariable>
+                        <SetVariable name="state_1_text">      °  &#9654; O  [ hide Channel mixer! ]</SetVariable>
```

Comment:
The general design in Tango is to only show big cover/spinny if channel mixer is collapsed.
@Be-ing pointed out that this option in skin settings (small or big spinny) wasn't clear if channel mixer is expanded.
I could make the label more explicit: `Hide channel mixer to choose big Cover or Spinny`
Do you consider the options `°` and `O` clear enough?